pkg/k8s/pod.go

```diff
@@ -15,7 +15,9 @@ import (
 )
 
 const (
-	podConditionTypeForUnhealthy = v1.PodReady
+	podConditionTypeForUnhealthy       = v1.PodReady
+	doNotDisruptAnnotation             = "cyclops.atlassian.com/do-not-disrupt"
+	doNotDisruptAnnotationRequredValue = "true"
```

Comment:
Typo in `doNotDisruptAnnotationRequredValue`, should be `doNotDisruptAnnotationRequiredValue`